modules/manually-configure-iam-nutanix.adoc

```diff
@@ -0,0 +1,140 @@
+= Configuring IAM for Nutanix
+
+Installing the cluster requires that the Cloud Credential Operator (CCO) operate in manual mode. While the installation program configures the CCO for manual mode, you must specify the identity and access management secrets.
+
+.Prerequisites
+
+* You have configured the `ccoctl` binary.
+
+* You have an `install-config.yaml` file.
+
+.Procedure
+
+. Create a YAML file that contains the credentials data in the following format:
++
+.Credentials data format
+[source,yaml]
+----
+credentials:
+- type: basic_auth <1>
+  data:
+    prismCentral: <2>
+      username: <username_for_prism_central>
+      password: <password_for_prism_central>
+    prismElements: <3>
+    - name: <name_of_prism_element>
+      username: <username_for_prism_element>
+      password: <password_for_prism_element>
+----
+<1> Specify the authentication type. Only basic authentication is supported.
+<2> Specify the Prism Central credentials.
+<3> Optional: Specify the Prism Element credentials.
+
+. Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc adm release extract --credentials-requests --cloud=nutanix \
+--to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+quay.io/<path_to>/ocp-release:<version>
+----
++
+<1> Specify the path to the directory that contains the files for the component `CredentialsRequests` objects. If the specified directory does not exist, this command creates it.
++
+.Sample `CredentialsRequest` object
+[source,yaml]
+----
+  apiVersion: cloudcredential.openshift.io/v1
+  kind: CredentialsRequest
+  metadata:
+    annotations:
+      include.release.openshift.io/self-managed-high-availability: "true"
+    labels:
+      controller-tools.k8s.io: "1.0"
+    name: openshift-machine-api-nutanix
+    namespace: openshift-cloud-credential-operator
+  spec:
+    providerSpec:
+      apiVersion: cloudcredential.openshift.io/v1
+      kind: NutanixProviderSpec
+    secretRef:
+      name: nutanix-credentials
+      namespace: openshift-machine-api
+----
+
+. Use the `ccoctl` tool to process all of the `CredentialsRequest` objects in the `credrequests` directory by running the following command:
++
+[source,terminal]
+----
+$ ccoctl nutanix create-shared-secrets \
+--credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+--output-dir=<ccoctl_output_dir> \ <2>
+--credentials-source-filepath=<path_to_credentials_file> <3>
+----
++
+<1> Specify the path to the directory that contains the files for the component `CredentialsRequests` objects.
+<2> Specify the directory that contains the files of the component credentials secrets, under the `manifests` directory. By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag.
```

Comment:
> I'm a bit confused about this directory, both here and later in the procedure. Are you trying to say that you specify a directory so this command can create a `manifests` directory in it?

@jeana-redhat Thoughts on Kathryn's comment?

Reply:
Hm, yes this could be better stated. It would be a change in multiple (already published) versions of this procedure for different platforms, so probably easier to handle in one go as a separate effort.